internal/services/containerapps/container_app_environment_resource_test.go

```diff
@@ -219,7 +219,7 @@ func TestAccContainerAppEnvironment_infraResourceGroup(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("workload_profile"),
```

Comment:
what's the reason for adding this as an importstep?

Reply:
As described above.